app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] As stated in the project requirement Make sure that your app handles the URLs used in the wireframes: So when I run the app to check the app handles the URLs used in the wireframes I got the error because of typically occurs when Rails cannot find a view template that matches the controller action. In your case, it’s looking for a template called `index.html.erb` in the app/views/users directory because you’re trying to access the index action of your `UsersController`. **This happen because of your project  structure of view directory in your project is incorrect which as below views/layouts/posts and views/layouts/users.** 
![image](https://github.com/JuanDiegoGuerra/Blog-App-RoR/assets/93598189/88afd3f9-1a4e-48ed-b046-79cb7b16258c)

![image](https://github.com/JuanDiegoGuerra/Blog-App-RoR/assets/93598189/ca0d3c3b-8601-4cd5-90a5-2936dde6ee4e)

- But the correct View directory structure should be views/layouts, views/users and views/posts Which run the app correctly So kindly Fix the issues 

![image](https://github.com/JuanDiegoGuerra/Blog-App-RoR/assets/93598189/294c23f4-de04-4f6d-9e06-cb5a8557d83a)

![Screenshot_1](https://github.com/JuanDiegoGuerra/Blog-App-RoR/assets/93598189/c08bb05e-516c-4c05-acb8-c0fa390529ac)

Reply:
- Big thanks for clearing me the problem. I'm not that used to receive this amazing feedback🤩. 
⭐**5**⭐